packages/grafana-ui/src/components/TimePicker/TimePicker.tsx

```diff
@@ -140,6 +157,12 @@ export class UnthemedTimePicker extends PureComponent<Props, State> {
     const { isOpen } = this.state;
     const styles = getStyles(theme);
     const hasAbsolute = isDateTime(value.raw.from) || isDateTime(value.raw.to);
+    const syncedTimePicker = timeSyncButton && isSynced;
+    const timePickerIconClass = classNames('fa fa-clock-o fa-fw', { ['icon-brand-gradient']: syncedTimePicker });
```

Comment:
Can we use Emotion's `cx` function instead of `classNames`?

Reply:
Of course! 🙂